Yafc.Model/Model/AutoPlanner.cs

```diff
@@ -23,7 +24,9 @@ public class AutoPlannerRecipe {
         public HashSet<Recipe> downstream = new HashSet<Recipe>();
         public HashSet<Recipe> upstream = new HashSet<Recipe>();
     }
+}
 
+namespace YAFC.Model {
```

Comment:
The whole file `AutoPlanner.cs` is inside the namespace `Yafc.Model`. I came to that conclusion after collapsing both namespaces and finding nothing else.

Could you please elaborate on why we need to wrap them in namespaces separately?

Reply:
Not the same namespace, they are case sensitive, and more specifically, the json serializer is case-sensitive too (see the TypeSerializer)

![afbeelding](https://github.com/have-fun-was-taken/yafc-ce/assets/4461459/82b02402-6c48-43e9-b52c-0d9d5a4a16ad)

Reply:
I see it now. Thanks!